Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM golang:1.14-alpine AS build
+
+ARG LDFLAGS
+
+# Install some dependencies needed to build the project
+RUN apk add bash ca-certificates git gcc g++ libc-dev
```

Comment:
```suggestion
```
None of these are required by the following commands.

Reply:
I removed the apk from the build